packages/integrations/cloudflare/src/index.ts

```diff
@@ -24,11 +24,31 @@ export function getAdapter(isModeDirectory: boolean): AstroAdapter {
 				name: '@astrojs/cloudflare',
 				serverEntrypoint: '@astrojs/cloudflare/server.directory.js',
 				exports: ['onRequest', 'manifest'],
+				supportedAstroFeatures: {
+					hybridOutput: 'Stable',
+					staticOutput: 'Unsupported',
+					serverOutput: 'Stable',
+					assets: {
+						supportKind: 'Unsupported',
+						isSharpCompatible: false,
+						isSquooshCompatible: false,
+					},
+				},
 		  }
 		: {
 				name: '@astrojs/cloudflare',
 				serverEntrypoint: '@astrojs/cloudflare/server.advanced.js',
 				exports: ['default'],
+				supportedAstroFeatures: {
+					hybridOutput: 'Stable',
+					staticOutput: 'Unsupported',
+					serverOutput: 'Stable',
+					assets: {
+						supportKind: 'Unsupported',
+						isSharpCompatible: false,
+						isSquooshCompatible: false,
```

Comment:
```suggestion
						supportKind: 'Stable',
						isSharpCompatible: false,
						isSquooshCompatible: false,
```

Every adapter supports `assets` the feature, what they don't support is the image services. Someone could write an image service powered by a supported library (or Cloudflare's Image API in this case)